File: discord-bot/commands/ping.js
const { MessageFlags } = require('discord.js');
const { SlashCommandBuilder } = require('discord.js');
const { simple } = require('../src/utils/embedBuilder');

module.exports = {
    data: new SlashCommandBuilder()
        .setName('ping')
        .setDescription('Replies with Pong!'),
    async execute(interaction) {
        const embed = simple('Pong!');
        await interaction.reply({ embeds: [embed], flags: [MessageFlags.Ephemeral] });
    },
};

File: ironaccord_bot/cogs/ping.py
import discord
from discord import app_commands
from discord.ext import commands

from ..utils.embed import simple

class PingCog(commands.Cog):
    def __init__(self, bot: commands.Bot):
        self.bot = bot

    @app_commands.command(name="ping", description="Replies with Pong!")
    async def ping(self, interaction: discord.Interaction):
        embed = simple("Pong!")
        await interaction.response.send_message(embed=embed, ephemeral=True)

async def setup(bot: commands.Bot):
    await bot.add_cog(PingCog(bot))
